Gaijinizer patch/Map172.txt

```diff
@@ -64,9 +65,9 @@ What does that mean to you...?#Triketos
  悪魔は\\C[2]『人間との約束を破れない』だけであって
 『人間に嘘をつけない』わけじゃない………
 > CONTEXT: Map172/events/2/pages/0/42/Dialogue/kazuya-b
-Yuuma ...... Don't you know? 
-Demons \\C[2]"cannot break promises with humans".
-It doesn't mean they can't lie.#Triketos
+Yuuma... Don't you know? 
+Just because demons \\C[2]cannot break promises with
+humans, doesn't mean they can't lie.#Triketos
 > END STRING
```

Comment:
Minor flow:
>_**Yuuma... don't you know?**_
>
>_**Just because demons \\\C[2]can't break promises with
humans, that doesn't mean they can't lie.**_